libs/ui-watt/project.json

```diff
@@ -41,8 +42,9 @@
       "executor": "@nrwl/storybook:build",
       "outputs": ["{options.outputPath}"],
       "options": {
+        "outputPath": "dist/designsystem",
+        "projectBuildConfig": "app-dh",
```

Comment:
suggestion: By looking at a spec file in Nx (https://github.com/nrwl/nx/blob/dcb839c49d102f5d50b5d54875750120c2090faf/packages/storybook/src/migrations/update-13-4-6/__snapshots__/set-project-build-config.spec.ts.snap#L109), it seems like `projectBuildConfig` should be set to something like `ui-watt:build-storybook`.